src/main/java/com/adoptmeplus/enterprise/AdoptMePlusController.java

```diff
@@ -48,6 +52,7 @@ public ResponseEntity fetchAdoptionsById(@PathVariable("id") String id){
      */
 
     @PostMapping(value="/adoptions/create", consumes="application/json", produces="application/json")
+    @ResponseBody
```

Comment:
https://www.baeldung.com/spring-request-response-body

The @ResponseBody annotation prevents the 500 error when posting a new adoption